<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PalindromeFinder</class>
 <widget class="QMainWindow" name="PalindromeFinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>201</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>251</width>
    <height>201</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>251</width>
    <height>201</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PalindromeFinder</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QTextBrowser" name="resultsBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Result Box</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>231</width>
       <height>32</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Search For Largest Palindrome</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="firstNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>First Number</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="secondNumber">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Second Number</string>
     </property>
    </widget>
   </widget>
   <zorder>widget</zorder>
   <zorder>gridLayoutWidget</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
